content/05.heterogeneity.md

```diff
@@ -1,18 +1,20 @@
+In practice, researchers often use the first few PCs to reduce the dimensionality without removing what may be important or informative variability in the data, though PCs are obtained without regard for labels (e.g., disease vs. control or dataset of origin).
+Beyond reducing the number of features in various types of data [@doi:10.1016/j.media.2020.101660; @doi:10.1038/ncomms14825], dimensionality reduction can also be used to visualize structure or artifacts in the data (e.g., [@doi:10.1038/s41467-019-13056-x]), to define sample subgroups (e.g., [@doi:10.1038/s41467-020-15351-4], or for feature selection and extraction during application of specific machine learning models [@doi:10.1007/978-3-030-03243-2_299-1] (Figure {@fig:1}D). 
 
-Methods like PCA, MDS, t-SNE, and UMAP can successfully identify the effect of these variables on the original data, though t-SNE and UMAP may require tuning of hyperparameters that may effect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
-Furthermore, testing multiple dimensionality reduction methods, rather than a single method, may be necessary to obtain a more comprehensive portrait of the data [@doi:10.1186/s13059-020-02021-3]. 
+Methods like PCA, multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can help researchers successfully identify useful patterns in the original data, though t-SNE and UMAP may require tuning of hyperparameters – values used to set up a model that are often specified by a user, rather than learned from the data – that may effect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
+Testing multiple dimensionality reduction methods, rather than a single method, may be necessary to obtain a more comprehensive portrait of the data [@doi:10.1186/s13059-020-02021-3]. 
 Nguyen and Holmes discuss additional important considerations for using dimensionality reduction methods such as selection criteria and interpretation of results [@doi:10.1371/journal.pcbi.1006907].
 Beyond dimensionality reduction, other unsupervised learning approaches such as k-means clustering or hierarchical clustering have been used to characterize the structure present in genomic or imaging data [@doi:10.1186/1471-2105-9-497; @doi:10.1109/jbhi.2013.2276766].
 
-Dimensionality reduction, or more fundamentally, representation learning, learns low-dimensional representations (composite features) from the raw data. 
+Representation learning approaches, of which dimensionality reduction methods are a subset, learns low-dimensional representations (composite features) from the raw data. 
```

Comment:
Response to comment https://github.com/jaybee84/ml-in-rd/issues/174#issuecomment-889202535

> `Dimensionality reduction, or more fundamentally, representation learning` is this implying that both dimensionality reduction and representation learning are the same?